modules/contact-form/css/grunion.css

```diff
@@ -1,5 +1,5 @@
 .contact-form .clear-form { clear: both; }
-.contact-form input[type='text'], .contact-form input[type='email'] { width: 300px; max-width: 98%; margin-bottom: 13px; }
+.contact-form input[type='text'], .contact-form input[type='email'], .contact-form input[type='date'] { width: 300px; max-width: 98%; margin-bottom: 13px; }
```

Comment:
We should probably break these lines up to have subsequent selectors and properties on subsequent lines -- it will also make diffs easier.  (as per wp core css style guide)